src/opera/pge/disp_s1/disp_s1_pge.py

```diff
@@ -589,6 +593,33 @@ def _qa_log_filename(self):
         """
         return self._ancillary_filename() + ".qa.log"
 
+    def augment_measured_parameters(self, measured_parameters):
+        descriptions_file = self.runconfig.iso_measured_parameter_descriptions
+
+        new_measured_parameters = {}
+
+        if descriptions_file is not None:
```

Comment:
Wondering if it makes more sense to just test `if descriptions_file`, so if somehow we're passed an empty string we still trigger the error case.